Assets/UniGLTF/Runtime/UniGLTF/IO/MaterialIO/RenderPipelineMaterialDescriptorUtility.cs

```diff
@@ -0,0 +1,45 @@
+using UniGLTF;
+using UnityEngine.Rendering;
+
+public class RenderPipelineMaterialDescriptorGeneratorUtility
+{
+    protected static RenderPipelineTypes GetRenderPipelineType()
+    {
+        RenderPipeline currentPipeline = RenderPipelineManager.currentPipeline;
+
+        if (currentPipeline == null)
+        {
+            return RenderPipelineTypes.Unknown;
```

Comment:
It seems to be `RenderPipelineTypes.BuiltinRenderPipeline` when `currentPipelin` is `null`.

Reply:
I understand that `currentPipeline == null` means current render pipeline is Built-in RenderPipeline in Unity.

But this implementation has different behaviour.
The loader can't decide `RenderPipelineTypes` correctly when Project's RenderPipeline is Built-in RenderPipeline.

Suggested change is below.

```suggestion
            return RenderPipelineTypes.BuiltinRenderPipeline;
```

---
How to run unit tests

Select `Window` -> `General` -> `Test Runner` then `Test Runner` Windows is opened.
In `Test Runner` Window, select `Run All` button then tests are running.
You can find failed tests in the list.

Reply:
Please see latest changes